sorc/link_workflow.sh

```diff
@@ -295,10 +295,30 @@ fi
 
 # GDASApp
 if [[ -d "${script_dir}/gdas.cd" ]]; then
-  for gdasexe in fv3jedi_addincrement.x fv3jedi_diffstates.x fv3jedi_ensvariance.x fv3jedi_hofx.x \
-    fv3jedi_var.x fv3jedi_convertincrement.x fv3jedi_dirac.x fv3jedi_error_covariance_training.x \
-    fv3jedi_letkf.x fv3jedi_convertstate.x fv3jedi_eda.x fv3jedi_forecast.x fv3jedi_plot_field.x \
-    fv3jedi_data_checker.py fv3jedi_enshofx.x fv3jedi_hofx_nomodel.x fv3jedi_testdata_downloader.py; do
+  declare -a JEDI_EXE=("fv3jedi_addincrement.x" \
+                       "fv3jedi_diffstates.x" \
+                       "fv3jedi_ensvariance.x" \
+                       "fv3jedi_hofx.x" \
+                       "fv3jedi_var.x" \
+                       "fv3jedi_convertincrement.x" \
+                       "fv3jedi_dirac.x" \
+                       "fv3jedi_error_covariance_training.x" \
+                       "fv3jedi_letkf.x" \
+                       "fv3jedi_convertstate.x" \
+                       "fv3jedi_eda.x" \
+                       "fv3jedi_forecast.x" \
+                       "fv3jedi_plot_field.x" \
+                       "fv3jedi_data_checker.py" \
+                       "fv3jedi_enshofx.x" \
+                       "fv3jedi_hofx_nomodel.x" \
+                       "fv3jedi_testdata_downloader.py" )
+            JEDI_EXE+=(
```

Comment:
Fine with putting these in a list (we should probably do the same for the others in this file), but why two separate declarations without any control structures making the appended bit conditional?